pages/api/utils/policy-utils.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const mapToClientModel = (policyResponse) => {
+  return {
+    id: policyResponse.id,
+    name: policyResponse.name,
+    description: policyResponse.description,
+    regoContent: policyResponse.policy.regoContent,
+  };
+};
+
+export const mapToApiModel = (request) => {
+  let formattedRequest = request.body;
+
+  if (
+    typeof request.body === "string" &&
```

Comment:
Is this still needed? I created and updated a policy and it looks like `req.body` is always being parsed now.

Reply:
oh yup, good call I will remove this now